docs/source/http_api/access.rst

```diff
@@ -99,12 +99,21 @@ A simple example in Python::
 
     import hashlib
     import hmac
-    import urllib
     import time
-
-
-    def build_indico_request(path, params, api_key=None, secret_key=None, only_public=False, persistent=False):
-        items = params.items() if hasattr(params, 'items') else list(params)
+    from future.moves.urllib.parse import urlencode
+    ## python 3
+    # from urllib.parse import urlencode
+    ## python 2
+    # from urllib import urlencode
+
+
+    def build_indico_request(path,
+                             params,
```

Comment:
I don't think wrapping it increases readability - and if someone pastes it in their code changes are good they'll reformat it according to their own codebase's style anyway.

Reply:
whichever you prefer, maitainer edits are enabled.